.github/workflows/tests.yml

```diff
@@ -0,0 +1,39 @@
+name: Tests
+
+on:
+  push:
```

Comment:
Do we need to build on push, or is PR enough? All code making its way into `master` will be via a PR, so perhaps we could remove this?

Reply:
@matthew-inamdar Yes, if you push to any branch it will run tests.

Reply:
Cool, mering now 👍